os/kernel/binary_manager/binary_manager_load.c

```diff
@@ -544,27 +515,34 @@ static int reloading_thread(int argc, char *argv[])
 		bmdbg("Terminate binary %d\n", bidx);
 	}
 
-	/* 2. Unload common binary */
+	/* Unload common binary */
```

Comment:
Could you let me know why common should be terminated later? If it does not need, let's merge it into the for loop.

Reply:
I will check and modify it.